src/index.njk

```diff
@@ -30,23 +30,32 @@ masthead: true
         <hr class="govuk-section-break govuk-section-break--visible govuk-section-break--xl govuk-!-margin-top-0">
       </div>
 
+      <div class="govuk-grid-column-two-thirds govuk-!-mb-r4">
+        <h2 class="govuk-heading-l">Principles we follow</h2>
+        <p class="govuk-body">
+The GOV.UK Design System helps service teams follow the <a class="govuk-link" href="https://www.gov.uk/guidance/government-design-principles">Government Design Principles</a> and <a class="govuk-link" href="https://www.gov.uk/service-manual">GOV.UK Service Manual</a>. This website also follows the Government Digital Service (GDS) <a class="govuk-link" href="https://www.gov.uk/guidance/style-guide">style guide</a> and <a class="govuk-link" href="https://www.gov.uk/guidance/content-design">content design guidance</a>.
```

Comment:
Will check on a question @timpaul brought up on whether we can call it the "GOV.UK style guide" rather than GDS style guide.

Reply:
Asked content-community and it doesn't sound like there's agreement to call it the "GOV.UK style guide". Calling it the GDS style guide came from hopes it would be applied more widely than just GOV.UK.

Instead of quibbling over branding, I've decided to go with the still-accurate "style guide... used by GOV.UK" instead.